kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -17,8 +17,8 @@
             <td>{{mtime}}</td>
             <td>
                 {{#if imported }}
-                <div class="pull-right" style="margin-left: -5px">
-                    <select id="import-type" style="width: 118px;">
+                <div class="pull-right" style="margin-left: -26px">
```

Comment:
could we give this div a specific class and then move the styling stuff into the css?